src/components/Head.vue

```diff
@@ -50,7 +50,7 @@ export default {
       this.showNetworks = false
     },
     async switchNetwork (network) {
-      await this.changeActiveNetwork({ network })
+      await this.changeActiveNetwork({ currentNetwork: this.activeNetwork, network })
```

Comment:
Why have to pass in `currentNetwork`?
You can retrieve current network before mutation in the action
```
export const changeActiveNetwork = async ({ commit, state }, {
  currentNetwork,
  network
}) => {
state.activeNetwork
```

Reply:
changed to get this value from the state inside the action